topostats/plottingfuncs.py

```diff
@@ -25,6 +25,31 @@
 # pylint: disable=dangerous-default-value
 
 
+def add_pixel_to_nm_to_plotting_config(plotting_config: dict, pixel_to_nm_scaling: float) -> dict:
+    """Function to ensure that the plotting config has the pixel to nanometre scaling factor accessible
```

Comment:
I'm probably being overly pedantic but I don't think docstrings need prefixing with "_Function to_".

```suggestion
    """Ensure that the plotting config has the pixel to nanometre scaling factor accessible
```